graphics.c

```diff
@@ -51,6 +52,11 @@ static unsigned char gr_current_a = 255;
 
 static GRSurface* gr_draw = NULL;
 
+minui_backend* open_adf()
+{
+    return 0;
+}
```

Comment:
Why?

Reply:
We don't build graphics_adf.c.  Declaring an alternative symbol which returns null avoids the need to rewrite the initialization code to remove the use of it in turn reducing the potential for merge conflicts when merging in upstream changes.